src/shape/bounding-sphere.js

```diff
@@ -3,6 +3,7 @@ pc.extend(pc, function () {
     var tmpVecB = new pc.Vec3();
     var tmpVecC = new pc.Vec3();
     var tmpVecD = new pc.Vec3();
+    var diffBetweenPoints = new pc.Vec3();
```

Comment:
`functionA` - this was not a suggestion, we never did named functions like so. As functions always have specific purpose.
In case with vectors - their purpose to "use and throw", so they have no meaning outside of context, that is why called "tmp".
We've used "tmp" approach all around in many many projects, and this been good so far. Lets think of future problems when they arrive in this case, as practice shows - this wont be a problem.